.gitignore

```diff
@@ -26,6 +26,12 @@
 .idea/**/gradle.xml
 .idea/**/libraries
 
+.idea/.gitignore
+.idea/modules.xml
+.idea/opensergo-control-plane.iml
+.idea/protoeditor.xml
+.idea/vcs.xml
+
```

Comment:
I think all files from IDE of `intelliJ IDEA` is unnecessary, so i think we can ignore them like follwing:

```
.idea/
.idea_modules/
*.iml
```

Reply:
> I think all files from IDE of `intelliJ IDEA` is unnecessary, so i think we can ignore them like follwing:
> 
> ```
> .idea/
> .idea_modules/
> *.iml
> ```

I think so,  I will modify the .gitignore